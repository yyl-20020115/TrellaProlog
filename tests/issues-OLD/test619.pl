:- op(300,xfx,\\).

main :-
	read_term_from_chars("[,@].", T).

:- initialization(main).
