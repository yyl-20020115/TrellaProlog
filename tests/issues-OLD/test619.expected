   error(syntax_error(args),read_term_from_chars/3).
